src/Tools/dotnet-user-secrets/src/dotnet-user-secrets.csproj

```diff
@@ -10,6 +10,7 @@
     <PackAsTool>true</PackAsTool>
     <!-- This package is for internal use only. It contains a CLI which is bundled in the .NET Core SDK. -->
     <IsShippingPackage>false</IsShippingPackage>
+    <ExcludeFromSourceBuild>false</ExcludeFromSourceBuild>
```

Comment:
How does this property work? Does it completely remove the project from consideration when building a solution from source? I'm wondering if this would allow us to remove SourceBuild.sln in Roslyn and instead rely on Compilers.sln for source build @333fred

Reply:
@jaredpar I think the answer to your questions can be found
- https://github.com/dotnet/arcade/blob/cdd5d318667c5af1740d69c8a7755e23e7964aeb/src/Microsoft.DotNet.Arcade.Sdk/sdk/Sdk.targets#L10
- https://github.com/dotnet/arcade/blob/cdd5d318667c5af1740d69c8a7755e23e7964aeb/src/Microsoft.DotNet.Arcade.Sdk/sdk/Sdk.targets#L25
- https://github.com/dotnet/arcade/blob/cdd5d318667c5af1740d69c8a7755e23e7964aeb/src/Microsoft.DotNet.Arcade.Sdk/tools/Empty.targets

I don't see a build job similar to this one in the Roslyn YAML, at least on your default branch. If you're building Roslyn using solutions on the CI, I suspect including SourceBuild.sln is just redundant.

Reply:
Perfect thanks. Yeah I think we can definitely delete SourceBuild.sln now. We do have it used in our YAML in a PR but not in a merged branch.